src/pages/my_page/user_profile/UserIdentity.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from 'react';
+import { Box, IconButton, TextField, Tooltip, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { BackgroundDiv } from '../../../components/styled/Wrappers';
+import { User } from '../../../types/user.types';
+import VisibilityOffIcon from '@mui/icons-material/VisibilityOff';
+import VisibilityIcon from '@mui/icons-material/Visibility';
+import { getMaskedNationalIdentityNumber } from '../../../utils/user-helpers';
+import { UserRoles } from './UserRoles';
+
+interface UserIdentitiesProps {
+  user: User;
+}
+
+export const UserIdentity = ({ user }: UserIdentitiesProps) => {
+  const { t } = useTranslation();
+  const fullName = `${user.givenName} ${user.familyName}`;
+  const userCristinId = user.cristinId?.split('/').at(-1);
+  const nationalId = user.nationalIdNumber;
+  const [showFullNin, setShowFullNin] = useState(false);
+
+  return (
+    <BackgroundDiv sx={{ bgcolor: 'secondary.dark' }}>
```

Comment:
secondary colors i theme er forskjøvet ett hakk når du henter fra develop neste gang
```suggestion
    <BackgroundDiv sx={{ bgcolor: 'secondary.main' }}>
```